### ISSUE TITLE
App crash if adding firebase Version 1.2.1

### ISSUE THREAD

**Issue by patriksharma** [travelpayouts/travel-app-android#59]
> Uncomment //apply plugin: 'com.google.gms.google-services' and //apply plugin: 'com.google.firebase.crashlytics' in app/build.gradle
> Uncomment //classpath 'com.google.gms:google-services:4.3.1' in build.gradle
> 

```

Gradle sync failed: Plugin with id 'com.google.firebase.crashlytics' not found.
				Consult IDE log for more details (Help | Show Log) (1 m 40 s 793 ms)
Doing above steps :  crashlytics is not working . So I have commented crashytlics and run app with google-services only
```

App build but doesn't start at all.

App working fine without firebase integration

**Comment by patriksharma** [comment]
Cross checked one more time. App keep crashing on Start when we integrate Firebase or google services.

App works fine if we comment firebase

**Comment by annaTP** [comment]
Hi,

Thanks for reporting the issue. We're testing the fix right now and will be releasing it ASAP.

**Comment by patriksharma** [comment]
resolved  with 1.2.2
